spec/qiita/markdown/processor_spec.rb

```diff
@@ -1268,6 +1312,33 @@
       end
     end
 
+    shared_examples_for "codepen" do |allowed:|
```

Comment:
`"codepen"` is inappropriate as the name of the shared examples, because codepen is allowed to use even though the parameter is `false`. I think it should be renamed as `"override codepen attributes"`.